website/source/docs/configuration/storage/s3.html.md

```diff
@@ -35,10 +35,11 @@ storage "s3" {
 
 - `endpoint` `(string: "")` – Specifies an alternative, AWS compatible, S3
   endpoint. This can also be provided via the environment variable
-  `AWS_DEFAULT_REGION`.
+  `AWS_S3_ENDPOINT`.
```

Comment:
Yep. Looks like it was copypasta. The code actually uses AWS_S3_ENDPOINT though.